sites/partners/src/listings/PaperListingForm/OpenHouseForm.tsx

```diff
@@ -0,0 +1,178 @@
+  const defaultValues = (() => {
+    if (!currentEvent) return null
+
+    const { startTime, endTime, url, note } = currentEvent || {}
+    const values = {}
+
+    url && Object.assign(values, { url })
+    note && Object.assign(values, { note })
+    startTime && Object.assign(values, { startTime: formatDateToTimeField(startTime) })
+    endTime && Object.assign(values, { endTime: formatDateToTimeField(endTime) })
+
+    if (startTime) {
+      const dateObj = moment(startTime)
+
+      const date = {
+        day: dateObj.format("DD"),
+        month: dateObj.format("MM"),
+        year: dateObj.format("YYYY"),
+      }
+
+      Object.assign(values, { date })
+    }
+
+    return values as OpenHouseFormValues
+  })()
+
+  // eslint-disable-next-line @typescript-eslint/unbound-method
+  const { register, watch, trigger, getValues, errors } = useForm<OpenHouseFormValues>({
+    defaultValues,
+  })
+
+  const handleSubmit = async () => {
+    const validation = await trigger()
+
+    if (validation) {
+      const data = getValues()
+
+      const event = {
+        ...currentEvent,
+        tempId: currentEvent.tempId ? currentEvent.tempId : nanoid(),
+        startTime: createTime(createDate(data.date), data.startTime),
+        endTime: createTime(createDate(data.date), data.endTime),
+        url: data.url,
+        note: data.note,
+      }
+
+      onSubmit(event)
+    }
+  }
+
+  return (
+    <Form onSubmit={() => false}>
+      <div className="border rounded-md p-8 bg-white">
+        <GridSection title={t("listings.sections.openHouse")} columns={3}>
+          <GridCell>
+            <ViewItem label={t("t.date")}>
+              <DateField
+                label={t("t.date")}
+                name="date"
+                id="date"
+                register={register}
+                watch={watch}
+                readerOnly
+                error={errors?.date}
+                errorMessage={t("errors.requiredFieldError")}
+                required
+                defaultDate={defaultValues?.date}
+              />
+            </ViewItem>
+          </GridCell>
+          <GridCell>
+            <ViewItem label={t("t.startTime")}>
+              <TimeField
+                label={t("t.startTime")}
+                name="startTime"
+                id="startTime"
+                register={register}
+                watch={watch}
+                readerOnly
+                error={!!errors?.startTime}
+                required
+                defaultValues={defaultValues?.startTime}
+              />
+            </ViewItem>
+          </GridCell>
+          <GridCell>
+            <ViewItem label={t("t.end")}>
+              <TimeField
+                label={t("t.end")}
+                name="endTime"
+                id="endTime"
+                register={register}
+                watch={watch}
+                readerOnly
+                error={!!errors?.startTime}
```

Comment:
maybe this is meant to be `errors?.endTime`

Reply:
Good catch! I've defined` !!errors?.endTime`, because we need to pass a boolean.